test/one/profiler/test/Output.java

```diff
@@ -62,12 +63,30 @@ public Output convertFlameToCollapsed() throws IOException {
         }
 
         try (ByteArrayOutputStream outputStream = new ByteArrayOutputStream();
-             PrintStream out = new PrintStream(outputStream)) {
+                PrintStream out = new PrintStream(outputStream)) {
```

Comment:
Please do not reformat untouched code, especially since the existing style already matches default IDEA settings.

Reply:
I was using vscode default rules, will not format, thanks.